examples/chatgpt/rag-quickstart/azure/Azure_AI_Search_with_Azure_Functions_and_GPT_Actions_in_ChatGPT.ipynb

```diff
@@ -0,0 +1,1330 @@
+                "        batch_client.flush()\n",
+                "        \n",
+                "        print(f\"Uploaded {len(documents)} documents in total\")\n",
+                "    else:\n",
+                "        print(\"No documents to upload.\")\n",
+                "except HttpResponseError as e:\n",
+                "    print(f\"An error occurred: {e}\")\n",
+                "    raise  # Re-raise the exception to ensure it errors out\n",
+                "finally:\n",
+                "    # Clean up resources\n",
+                "    batch_client.close()\n"
+            ]
+        },
+        {
+            "attachments": {},
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "## Perform a vector similarity search\n",
+                "To test if the documents uploaded successfully, you can run the below code to see if it found different documents based on the `query` and `category`"
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {},
+            "outputs": [],
+            "source": [
+                "# Pure Vector Search\n",
+                "query = \"What model should I use to embed?\"\n",
+                "# Note: we'll have the GPT choose the category automatically once we put it in ChatGPT\n",
+                "category =\"models\"\n",
+                "credential = DefaultAzureCredential()\n",
+                "search_client = SearchClient(search_service_endpoint, index_name, AzureKeyCredential(search_service_api_key))\n",
+                "vector_query = VectorizedQuery(vector=generate_embeddings(query, embeddings_model), k_nearest_neighbors=3, fields=\"content_vector\")\n",
+                "  \n",
+                "results = search_client.search(  \n",
+                "    ## Notice None is query below, meaning we are only searching on the vectory query and not a keyword search on the query iteself\n",
+                "    search_text=None,  \n",
+                "    vector_queries= [vector_query], \n",
+                "    select=[\"title\", \"text\"],\n",
+                "    filter=f\"category eq '{category}'\"\n",
+                ")\n",
+                "\n",
+                "    # print(f\"URL: {result['url']}\\n\")  \n",
+                "for result in results:  \n",
+                "    print(result)\n"
+            ]
+        },
+        {
+            "attachments": {},
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "## Perform a Hybrid Search\n",
+                "Hybrid search combines the capabilities of traditional keyword-based search with vector-based similarity search to provide more relevant and contextual results. \n",
+                "\n",
+                "The provided code snippet demonstrates how to execute a hybrid search query:"
```

Comment:
It looks like the only difference between hybrid search and vector similarity is this single parameter; can we consolidate these sections and call this out?